client/src/common/client.ts

```diff
@@ -1739,6 +1742,7 @@ export abstract class BaseLanguageClient implements FeatureClient<Middleware, La
 		this.registerFeature(new InlayHintsFeature(this));
 		this.registerFeature(new DiagnosticFeature(this));
 		this.registerFeature(new NotebookDocumentSyncFeature(this));
+		this.registerFeature(new InlineCompletionItemFeature(this));
```

Comment:
Actually I overlooked that in the first review. We in general first add new features as proposed even if they are already final in VS Code since we need to allow the LSP community to provide feedback as well.

So we need to add the feater first in `ProposedFeatures.createAll` and a client needs to call `registerProposedFeatures` to make use of it.

Reply:
Sorry for not noticing this earlier.

Reply:
No worries. Done.